GoProjects/CalorieCounter/main.go

```diff
@@ -7,13 +7,9 @@ import (
 
 func main() {
 	var p food.Plate
```

Comment:
Write a function called testFunction and add the code from main into it then call that function from main.